src/main/java/mate/academy/model/Car.java

```diff
@@ -0,0 +1,74 @@
+package mate.academy.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car(String model, Manufacturer manufacturer, List<Driver> drivers) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.drivers = drivers;
```

Comment:
Although Car model has List<Driver> drivers field, it will not receive list from outside. Don’t add drivers to the constructors’ arguments - initialize drivers as an empty List inside the constructor instead.
це в чеклісті зазначено

Reply:
\+

Reply:
fix